crates/client/assets/schema.sdl

```diff
@@ -762,6 +762,7 @@ type NodeInfo {
 	maxTx: U64!
 	maxDepth: U64!
 	nodeVersion: String!
+	currentPoolGas: U64!
```

Comment:
It would be nice to expose all information, number of transactions, used gas, used size

Reply:
Okey done :)